libraries/script-engine/src/ScriptEngine.cpp

```diff
@@ -4,2841 +4,64 @@
-    registerGlobalObject("Vec3", &_vec3Library);
-    registerGlobalObject("Mat4", &_mat4Library);
-    registerGlobalObject("Uuid", &_uuidLibrary);
-    registerGlobalObject("Messages", DependencyManager::get<MessagesClient>().data());
-    registerGlobalObject("File", new FileScriptingInterface(this));
-    registerGlobalObject("console", &_consoleScriptingInterface);
-    registerFunction("console", "info", ConsoleScriptingInterface::info, currentContext()->argumentCount());
-    registerFunction("console", "log", ConsoleScriptingInterface::log, currentContext()->argumentCount());
-    registerFunction("console", "debug", ConsoleScriptingInterface::debug, currentContext()->argumentCount());
-    registerFunction("console", "warn", ConsoleScriptingInterface::warn, currentContext()->argumentCount());
-    registerFunction("console", "error", ConsoleScriptingInterface::error, currentContext()->argumentCount());
-    registerFunction("console", "exception", ConsoleScriptingInterface::exception, currentContext()->argumentCount());
-    registerFunction("console", "assert", ConsoleScriptingInterface::assertion, currentContext()->argumentCount());
-    registerFunction("console", "group", ConsoleScriptingInterface::group, 1);
-    registerFunction("console", "groupCollapsed", ConsoleScriptingInterface::groupCollapsed, 1);
-    registerFunction("console", "groupEnd", ConsoleScriptingInterface::groupEnd, 0);
-
-    qScriptRegisterMetaType(this, animVarMapToScriptValue, animVarMapFromScriptValue);
-    qScriptRegisterMetaType(this, resultHandlerToScriptValue, resultHandlerFromScriptValue);
-
-    // Scriptable cache access
-    auto resourcePrototype = createScriptableResourcePrototype(qSharedPointerCast<ScriptEngine>(sharedFromThis()));
-    globalObject().setProperty("Resource", resourcePrototype);
-    setDefaultPrototype(qMetaTypeId<ScriptableResource*>(), resourcePrototype);
-    qScriptRegisterMetaType(this, scriptableResourceToScriptValue, scriptableResourceFromScriptValue);
-
-    // constants
-    globalObject().setProperty("TREE_SCALE", newVariant(QVariant(TREE_SCALE)));
-
-    registerGlobalObject("Assets", _assetScriptingInterface);
-    registerGlobalObject("Resources", DependencyManager::get<ResourceScriptingInterface>().data());
-
-    registerGlobalObject("DebugDraw", &DebugDraw::getInstance());
-
-    registerGlobalObject("Model", new ModelScriptingInterface(this));
-    qScriptRegisterMetaType(this, meshToScriptValue, meshFromScriptValue);
-    qScriptRegisterMetaType(this, meshesToScriptValue, meshesFromScriptValue);
-
-    registerGlobalObject("UserActivityLogger", DependencyManager::get<UserActivityLoggerScriptingInterface>().data());
-
-#if DEV_BUILD || PR_BUILD
-    registerGlobalObject("StackTest", new StackTestScriptingInterface(this));
-#endif
-
-    globalObject().setProperty("KALILA", "isWaifu");
-    globalObject().setProperty("Kute", newFunction([](QScriptContext* context, QScriptEngine* engine) -> QScriptValue {
```

Comment:
I have preserved as much functionality as I could to maintain compatibility with existing scripts and expectations, so don't worry this code has moved and was not deleted.

Reply:
I mean I actually think these two should be removed?

Reply:
Possibly.  I didn't make that decision part of this change though.

Reply:
I vote just remove them as part of this PR

Reply:
Something I did briefly consider and am willing to do with enough "votes".  If you notice this PR is complicated enough that I've tended to err on the side of preserving as much existing functionality as seemed good to do rather than sweeping other stuff up.  But... might be okay here.

Reply:
okay, dropping